rust/worker/src/storage/mod.rs

```diff
@@ -82,7 +82,7 @@ impl Storage {
     pub(crate) async fn put_file(&self, key: &str, path: &str) -> Result<(), PutError> {
         match self {
             Storage::S3(s3) => s3
-                .put_file(key, path)
+                .put_file(key, path, None)
```

Comment:
would propagating the same `part_size_bytes: Option<u64>` interface change to this make sense? wondering because the part size doesn't really have a meaning when using the local storage adapter

Reply:
Thats fine IMO, the local storage is only for testing, so the interface it provides can willfully do suboptimal / dumb things.